src/libdrakvuf/Makefile.am

```diff
@@ -105,7 +105,7 @@
 h_sources = libdrakvuf.h json-profile.h json-util.h os.h private.h vmi.h \
             win.h win-error-codes.h win-exports.h win-handles.h win-offsets.h \
             win-offsets-map.h win-wow-offsets.h win-wow-offsets-map.h \
-            ntstatus.h \
+            ntstatus.h filesystem.hpp\
```

Comment:
libdrakvuf is strictly a C library - no `hpp` please, if there is C++ magic in the header, please move the header into `plugins`

Reply:
Should I move it to the general plugins folder or to the hyperbee folder?

Reply:
Plugins folder, please :)

Reply:
I guess then I should include filesystem.hpp in src/plugins/Makefile.am as well. Below you see the way I included filesystem.hpp to the Makefile. It works, but is the place where I added it correct?
```
if PLUGIN_HYPERBEE
sources += hyperbee/hyperbee.cpp
sources += hyperbee/hyperbee.h
sources += filesystem.hpp
endif
```

As an alternative, I also could add it somewhere here?

```
sources += plugins.cpp plugins.h plugins_ex.cpp plugins_ex.h plugin_utils.cpp plugin_utils.h private.h type_traits_helpers.h
sources += output_format.h output_format/common.h output_format/csvfmt.h output_format/deffmt.h output_format/jsonfmt.h output_format/kvfmt.h
sources += output_format/xfmt.h output_format/ostream.h output_format/ostream.cpp
```

What do you prefer?

Reply:
If its meant to be sharable by multiple plugins than the latter

Reply:
@icedevml @BonusPlay Do you have a recommendation? The filsesystem.hpp thing was a PR from you and I don't have much opinions about that. What do you prefer? Do you want to use that in other plugins?

Reply:
https://github.com/tklengyel/drakvuf/pull/1133#discussion_r577090514

Reply:
Ok, I assume this means I should add this in general. Is pushed as well.